lib/elasticsearch/drain/autoscaling.rb

```diff
@@ -30,6 +31,9 @@ def find_instances_in_asg
         @instance_ids = instances
       end
 
+      # Describe an AutoScaling Group
```

Comment:
Looks like this comment belongs with the `describe_autoscaling_group` method. Unless I'm just reading the diff wrong?